.github/workflows/check-query-files-and-compilation.yml

```diff
@@ -47,12 +47,12 @@ jobs:
       - name: Compile parsers Unix like
         if: matrix.os != 'windows-2022'
         run: |
-          nvim --headless -c "TSInstallSync all" -c "q"
+          nvim --headless -c "TSInstallSync! all" -c "q"
 
       - name: Compile parsers Windows
         if: matrix.os == 'windows-2022'
         run: |
-          Neovim\\bin\\nvim.exe --headless -c "TSInstallSync all" -c "q"
+          Neovim\\bin\\nvim.exe --headless -c "TSInstallSync! all" -c "q"
```

Comment:
necessary to test our C parser, not the built-in one.

Reply:
another solution would be to use the "true" runtime file in tests. But the actual C parser would be better.